Fitness/Views/GymFacilitiesView.swift

```diff
@@ -0,0 +1,483 @@
+                } else {
+                    make.height.equalTo(42)
+                }
+            }
+            
+            basketballSeparator!.snp.updateConstraints {make in
+                make.top.equalTo(basketballTableView!.snp.bottom).offset(16)
+                make.leading.equalToSuperview().offset(24)
+                make.trailing.equalToSuperview().offset(-24)
+                make.height.equalTo(1)
+            }
+            
+            bowlingTableView!.snp.updateConstraints {make in
+                make.width.centerX.equalToSuperview()
+                make.top.equalTo(basketballSeparator!.snp.bottom).offset(16)
+                if (facilityData[1].isDropped) {
+                    make.height.equalTo(260)
+                } else {
+                    make.height.equalTo(42)
+                }
+            }
+            
+            poolSeparator!.snp.updateConstraints {make in
+                make.top.equalTo(bowlingTableView!.snp.bottom).offset(16)
+                make.leading.equalToSuperview().offset(24)
+                make.trailing.equalToSuperview().offset(-24)
+                make.height.equalTo(1)
+            }
+            
+            poolTableView!.snp.updateConstraints {make in
+                make.width.centerX.bottom.equalToSuperview()
+                make.top.equalTo(poolSeparator!.snp.bottom).offset(16)
+                if (facilityData[2].isDropped) {
+                    make.height.equalTo(260)
+                } else {
+                    make.height.equalTo(42)
+                }
+            }
+        case .teagle:
+            poolTableView!.snp.updateConstraints {make in
+                make.width.centerX.bottom.top.equalToSuperview()
+                if (facilityData[0].isDropped) {
+                    make.height.equalTo(260)
+                } else {
+                    make.height.equalTo(42)
+                }
+            }
+            
+        default:
+            return
+        }
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    @objc func dropFacilityHours(sender: FacilitiesGestureRecognizer) {
+        guard let unwrappedFacilityName = self.facilityName else {
+            return
+        }
+        
+        switch unwrappedFacilityName {
+        case .helenNewman:
+            if sender.title == "basketball" {
+                basketballTableView!.beginUpdates()
+                var modifiedIndices: [IndexPath] = []
+                for i in 0..<7 {
+                    modifiedIndices.append(IndexPath(row: i, section: 0))
+                }
+                
+                if (facilityData[0].isDropped) {
+                    facilityData[0].isDropped = false
+                    basketballTableView!.deleteRows(at: modifiedIndices, with: .fade)
+                    (basketballTableView!.headerView(forSection: 0) as! FacilityHoursHeaderView).downArrow.image = .none
```

Comment:
Is it more efficient to set the images initially and just toggle the `isHidden` attribute?